pkg/k8s/api.go

```diff
@@ -78,10 +77,10 @@ func (kubeapi *kubernetesApi) checkApiAccess(client *http.Client) *healthcheckPb
 		return checkResult
 	}
 
-	endpointToCheck, err := generateBaseKubernetesApiUrl(kubeapi.apiSchemeHostAndPort)
+	endpointToCheck, err := generateBaseKubernetesApiUrl(kubeapi.Config.Host)
```

Comment:
Since `Config` is an embedded struct, you can call `kubeapi.Host` directly here. Same goes for the rest of the `kubeapi.Config.Host` calls in this file.

Reply:
woah, that's cool! thanks!